employee-rostering-backend/src/main/java/org/optaweb/employeerostering/service/spot/SpotService.java

```diff
@@ -57,7 +57,7 @@ public Boolean deleteSpot(Integer tenantId, Long id) {
         Optional<Spot> spotOptional = spotRepository.findById(id);
 
         if (!spotOptional.isPresent()) {
-            throw new EntityNotFoundException("No Spot entity found with ID (" + id + ").");
```

Comment:
Why don't you throw an Exception?

Reply:
I initially thought that `false` should be returned when entity is not deleted (which includes the case when it's not found), but throwing an Exception for that does make more sense in hindsight. Will fix this to keep consistent with old code.